terraform/environments/analytical-platform-compute/cluster/eks-cluster.tf

```diff
@@ -88,51 +88,51 @@ module "eks" {
     "karpenter.sh/discovery" = local.eks_cluster_name
   }
 
-  eks_managed_node_group_defaults = {
-    ami_release_version = local.environment_configuration.eks_node_version
-    ami_type            = "BOTTLEROCKET_x86_64"
-    platform            = "bottlerocket"
-    metadata_options = {
-      http_endpoint               = "enabled"
-      http_put_response_hop_limit = 1
-      http_tokens                 = "required"
-      instance_metadata_tags      = "enabled"
-    }
-
-    block_device_mappings = {
-      xvdb = {
-        device_name = "/dev/xvdb"
-        ebs = {
-          volume_size           = 100
-          volume_type           = "gp3"
-          iops                  = 3000
-          throughput            = 150
-          encrypted             = true
-          kms_key_id            = module.eks_ebs_kms.key_arn
-          delete_on_termination = true
-        }
-      }
-    }
-
-    iam_role_additional_policies = {
-      AmazonEC2ContainerRegistryReadOnly = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
-      AmazonSSMManagedInstanceCore       = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
-      CloudWatchAgentServerPolicy        = "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
-      ECRPullThroughCachePolicy          = module.ecr_pull_through_cache_iam_policy.arn
-      EKSClusterLogsKMSAccessPolicy      = module.eks_cluster_logs_kms_access_iam_policy.arn
-    }
-
-    node_repair_config = {
-      enabled = true
-    }
-  }
-
   eks_managed_node_groups = {
     general = {
       min_size       = 1
       max_size       = 10
       desired_size   = 3
       instance_types = ["m6a.xlarge"]
+
+      use_latest_ami_release_version = false
+      ami_release_version            = local.environment_configuration.eks_node_version
+      ami_type                       = "BOTTLEROCKET_x86_64"
+      platform                       = "bottlerocket"
+      enable_monitoring              = true
+      metadata_options = {
+        http_endpoint               = "enabled"
+        http_put_response_hop_limit = 1
+        http_tokens                 = "required"
+        instance_metadata_tags      = "enabled"
+      }
+
+      block_device_mappings = {
+        xvdb = {
+          device_name = "/dev/xvdb"
+          ebs = {
+            volume_size           = 100
+            volume_type           = "gp3"
+            iops                  = 3000
+            throughput            = 150
+            encrypted             = true
+            kms_key_id            = module.eks_ebs_kms.key_arn
+            delete_on_termination = true
+          }
+        }
+      }
+
+      iam_role_additional_policies = {
+        AmazonEC2ContainerRegistryReadOnly = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
```

Comment:
this can be removed, its already added on nodes https://github.com/terraform-aws-modules/terraform-aws-eks/blob/30664d261760ec97c670f90df3c793431cb096fb/modules/eks-managed-node-group/main.tf#L581